Makefile

```diff
@@ -317,11 +317,11 @@ ifeq ($(CUSTOM_EMBED),false)
 	$(HOST_BUILD_DIR)/crc-embedder download --goos=$* $(EMBED_DOWNLOAD_DIR)
 endif
 
-macos-universal-binary: macos-release-binary $(TOOLS_BINDIR)/makefat
+$(BUILD_DIR)/macos-universal/crc: $(BUILD_DIR)/macos-arm64/crc $(BUILD_DIR)/macos-amd64/crc $(TOOLS_BINDIR)/makefat
```

Comment:
In other words, the end result is to rename `macos-universal-binary` to `macos-release-binary`

Reply:
yeah now that you mention it, i see it too!